### PR TITLE
Rewrite debug docs

### PR DESCRIPTION
**Pull request by cewbdex**
Please review and merge eventually.

### CODE REVIEW

File: docs/debugging.md [hunk 1]
```diff
@@ -1,15 +1,80 @@
...
+```bash title="Restarting Evok service"
+sudo systemctl restart evok
+```
+
+### Reading the logs
+
+To read the logs from Evok in permanent debug mode, you can use following command. It also prints information about other related services.
+
+```bash title="Viewing the Evok log"
+journalctl -e -u evok -u unipi-one-modbus
```
Comment (kratochvil01) [inline docs/debugging.md:66]: Remove unipi-one-modbus. This service running only on unipi 1.1

**Review by kratochvil01 — CHANGES_REQUESTED** [review]

File: docs/debugging.md (hunk 1 above)
Comment (cewbdex) [reply docs/debugging.md:66]: Yes, I know, but it does not throw any error, when the service is not present, so I think of this as a feature, not a bug. Provides more debug info this way, what do you think of my solution with this in mind?

Comment (kratochvil01) [reply docs/debugging.md:66]: unipi-one-modbus has nothing to do with Evok. It is a HW to Modbus TCP converter just like the unipitcp service for other unipi controllers. This service is not related to the tuning of the Evok as such.

Comment (cewbdex) [reply docs/debugging.md:66]: Thus, it might provide relevant information that would help the debugging. But I see your point.

**Review by kratochvil01 — APPROVED** [review]

**Pull request merged**